### ISSUE TITLE
How do I install this?

### ISSUE THREAD

**Issue by Mavielor** [MKhayle/XIVComboPlugin#337]
I have XIVLauncher but it says outdated on dalamund so it doesnt let me install. I downloaded the files here on github but I dont know what folder should I move them to

**Comment by jsliu013** [comment]
Currently it is in testing, you can access it thru dalamud testing system.
Be warned some jobs and tweaks may be broken, all you can do is wait or help with reporting bugs.

If you can wait, wait til the stable. If not, deal with the bugs.

**Comment by tristin68** [comment]
I downloaded the latest zip and copied the pathing into the experimental tab but it does not seem to activate and it does not show up in my dalmund plugin list

**Comment by jsliu013** [comment]
u don't need to do anything like that, u turn on testing/beta mode in dalamud settings then u can right click on xivcombo and click receive beta updates

Warning: there are a lot of plugins that are in testing and will possibly crash your game. test at your own risk and by doing so, you accept the risks

**Comment by Mavielor** [comment]
Yes you enable tasting builds then search for XIVCombo on plugins. Then you right click XIVCombo and click download testing build or something. I figured it out eventually and its working without problems so far.
